Cargo.toml

```diff
@@ -19,7 +19,10 @@ aws_lambda_events = "0.4"
 # Reference issue https://github.com/awslabs/aws-lambda-rust-runtime/issues/274 
 # for the reason to move away from official awslabs/aws-lambda-rust-runtime
 # in favor of using netlify's fork lamedh-dev/aws-lambda-rust-runtime
-netlify_lambda_http = "0.2"
+lamedh_http = "0.3"
+lamedh_runtime = "0.3"
```

Comment:
Any specific reason why the `lamed_runtime` crate is required as well?

Reply:
```suggestion
```
`lamedh_http` depends on `lamedh_runtime` and there is no direct dependency for `warp_lambda` on `lamedh_runtime` we could safely remove `lamedh_runtime`.

Reply:
This was left over from trying to use the updated `lambda_http` crate that the `lamedh_http` crate is forked from. There were enough differences between the crates that I decided to keep with the Netlify fork and must have forgotten that the `_runtime` crate isn't explicitly necessary with the Netlify fork.